sunpy/map/mapsequence.py

```diff
@@ -511,3 +511,24 @@ def all_meta(self):
         Return all the meta objects as a list.
         """
         return [m.meta for m in self.maps]
+
+    def save(self, filepath, filetype='auto', **kwargs):
+        """
+        Saves the maps in the map sequence to disk.
+        Each map from the sequence will be saved separately.
+
+        Currently this only supports saving files in the FITS format.
+
+        Parameters
+        ----------
+        filepath : str
+            Location to save file to.
```

Comment:
This is unclear whether this is supposed to be a directory name or a filename stub.